### ISSUE TITLE
Github doesn't render latex

### ISSUE THREAD

**Issue by GalRogozinski** [iotaledger/IOTA-2.0-Research-Specifications#1]
I have noticed that latex is used (in the mana spec for example)
But it is not rendered..

There are plugins that can solve the issue, but I would recommend just moving away from latex

**Comment by JanOonk** [comment]
Yep 7 months later equations are still broken, links are broken, section numbers are broken (see for example https://github.com/iotaledger/IOTA-2.0-Research-Specifications/blob/main/5.3%20Mana.md )

**Comment by JanOonk** [comment]
Here's a solution to render Latex in Markdown: https://stackoverflow.com/a/45603559/2386040


For example put this in your Markdown file:
![equation](http://latex.codecogs.com/gif.latex?O_t%3D%5Ctext%20%7B%20Onset%20event%20at%20time%20bin%20%7D%20t)
`
![equation](http://latex.codecogs.com/gif.latex?s%3D%5Ctext%20%7B%20sensor%20reading%20%7D) 
![equation](http://latex.codecogs.com/gif.latex?P%28s%20%7C%20O_t%20%29%3D%5Ctext%20%7B%20Probability%20of%20a%20sensor%20reading%20value%20when%20sleep%20onset%20is%20observed%20at%20a%20time%20bin%20%7D%20t)
`

**Comment by JanOonk** [comment]
A solution can be found here: https://stackoverflow.com/a/45603559/2386040

A made an example from this, which also works with different themes for example Dark theme:
https://github.com/JanOonk/LatextTest/blob/main/README.md
